init.go

```diff
@@ -17,6 +17,14 @@ var (
 	// EnvLogLevel is key of the environment variable that can be used to set the log
 	// level on Init.
 	EnvLogLevel = "SRC_LOG_LEVEL"
+	// EnvLogScopeLevel is key of the environment variable that can be used to
+	// override the log level for specific scopes and its children.
+	//
+	// It has the format "SCOPE_0=LEVEL_0,SCOPE_1=LEVEL_1,...".
+	//
+	// Note: these levels do not respect the root level (SRC_LOG_LEVEL), so this
+	// allows operators to turn up the verbosity of specific logs.
+	EnvLogScopeLevel = "SRC_LOG_SCOPE_LEVEL"
```

Comment:
We should note here that the scope must be the absolute scope, e.g. if you have:

```
l := log.Scoped("foo", "")
l2 := log.Scoped("bar", "")
```

You cannot use `SCOPE_bar=debug` to unmute `l2`

Reply:
We should also note this only affects `outputcore` - it does not affect what log entries other cores (like the Sentry core) receive

Reply:
> We should note here that the scope must be the absolute scope, e.g. if you have:

Do you think the use of the word "specific scopes and its children" is not clear enough? Maybe some examples would be useful?

> ```
> l := log.Scoped("foo", "")
> l2 := log.Scoped("bar", "")
> ```
>
> You cannot use `SCOPE_bar=debug` to unmute `l2`

I'm note sure I understand. bar=debug would unmute l2?

> We should also note this only affects `outputcore` - it does not affect what log entries other cores (like the Sentry core) receive

Added a note.

Reply:
> Do you think the use of the word "specific scopes and its children" is not clear enough? Maybe some examples would be useful?

Some examples may be nice! I think a potential misunderstanding might come from creating a sub scope and thinking that sub scope is the scope mentioned here

> I'm note sure I understand. bar=debug would unmute l2?

Ah, my bad - the example was supposed to be: 

```
l := log.Scoped("foo", "")
l2 := l.Scoped("bar", "")
```

In this case one might think the scope of l2 is bar, when it’s actually foo.bar